ci-operator/step-registry/ipi/conf/aws/custom-security-groups/ipi-conf-aws-custom-security-groups-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: ipi-conf-aws-custom-security-groups
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
+  grace_period: 10m
+  commands: ipi-conf-aws-custom-security-groups-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: ENABLE_SHARED_PHZ
```

Comment:
@gpei `ENABLE_SHARED_PHZ` is not required for this step, we can remove it.

Reply:
ok, let me do it